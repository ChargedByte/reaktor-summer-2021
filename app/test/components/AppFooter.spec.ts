import { createLocalVue, mount } from '@vue/test-utils'

import Vuetify from 'vuetify'

import AppFooter from '~/components/AppFooter.vue'

describe('AppFooter', () => {
  const localVue = createLocalVue()
  let vuetify: Vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  test('is a Vue instance', () => {
    const wrapper = mount(AppFooter, { localVue, vuetify })
    expect(wrapper.vm).toBeTruthy()
  })

  test('renders correctly', () => {
    const wrapper = mount(AppFooter, { localVue, vuetify })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
