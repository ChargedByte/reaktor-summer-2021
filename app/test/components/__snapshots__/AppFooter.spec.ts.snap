// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter renders correctly 1`] = `
"<footer class=\\"v-footer v-sheet theme--light v-footer--fixed\\" style=\\"left: 0px; right: 0px; bottom: 0px;\\">
  <div class=\\"spacer\\"></div> <a href=\\"https://github.com/ChargedByte/WiM\\" target=\\"_blank\\" class=\\"v-btn v-btn--icon v-btn--round theme--light v-size--small\\"><span class=\\"v-btn__content\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-github theme--light\\"></i></span></a>
</footer>"
`;
